join/free_trial.md

```diff
@@ -1,59 +1,71 @@
 
 # Registration
 
-[Click here](https://sso.terrascope.be/auth/realms/terrascope/protocol/openid-connect/auth?client_id=openeoplatform&redirect_uri=https://portal.terrascope.be/dashboard&state=0%2F95954a95-1968-4a64-8b88-fef0f47936fb&response_type=code&scope=openid) to register with the platform and immediately start a 30 day free trial. 
+Register for openEO platform and immediately start a 30 day free trial:
 
-Registration to the platform and management of your user account happens via EOPlaza ([portal.terrascope.be](https://portal.terrascope.be)) and [EGI Check-in](https://www.egi.eu/services/check-in/). The registration link takes you through 2 steps that are described in detail below. After registration, you are redirected to the [EOPlaza dashboard](https://portal.terrascope.be/dashboard) where you can manage your account.
+<a href="https://sso.terrascope.be/auth/realms/terrascope/protocol/openid-connect/auth?client_id=openeoplatform&redirect_uri=https://portal.terrascope.be/dashboard&state=0%2F95954a95-1968-4a64-8b88-fef0f47936fb&response_type=code&scope=openid" class="action-button" style="display: inline-block; font-size:1.2rem; color: #fff; background-color: #335e6f; padding: 0.8rem 1.6rem; border-radius: 4px; border-bottom: 1px solid #2e5564;">&rarr; Register for openEO Platform</a>
```

Comment:
@m-mohr I did this button look with inline CSS based on the buttons from https://docs.openeo.cloud/ .
Feel free to suggest a cleaner way to do this with existing vuepress components (I don't know vuepress well enough to figure this out)

Reply:
We could create a component and embed it, but for a single link it's not worth the time. If we need it more often, we can still do it.

Reply:
@soxofaan Quickly checked openeo.org... I don't have the time to test, but could you check whether 

`<NavLink class="action-button" :item="{link: 'https://sso.terrascope.be/auth/realms/terrascope/protocol/openid-connect/auth?client_id=openeoplatform&redirect_uri=https://portal.terrascope.be/dashboard&state=0%2F95954a95-1968-4a64-8b88-fef0f47936fb&response_type=code&scope=openid', text: '→ Register for openEO Platform'}"/>`

works instead of the `<a href... ></a>`?

Reply:
I indeed tried something like that, but it doesn't seem to work: the `<NavLink >` element ends up in the final HTML as-is and doesn't render anything (being invalid HTML)

Reply:
Okay, there's likely an issue with imports or so, but then let's stick to HTML for now.